src/components/Board/Board.js

```diff
@@ -0,0 +1,78 @@
+import { View, Text, Dimensions } from "react-native";
+import React from "react";
+import useChessBoard from "./hooks/useChessBoard";
+import Cell from "../Cell/Cell";
+
+import styles from "./styles";
+
+const windowWidth = Dimensions.get("window").width;
+
+const preset = {
```

Comment:
such constants should not be with code and should be extracted to separate files